pyresample/ewa/__init__.py

```diff
@@ -67,4 +67,85 @@
 
 """
 
+import logging
+import numpy as np
+from pyresample.ewa import _ll2cr, _fornav
+
+LOG = logging.getLogger(__name__)
+
+
+def ll2cr(swath_def, area_def, fill=np.nan, **kwargs):
```

Comment:
Eh I suppose. I put in kwargs when I wasn't sure what I needed to customize. I can change it.